python/test/benchmarking/bench_python_cpp_algos.py

```diff
@@ -35,6 +35,15 @@ def time_block(run_name, time_data):
     time_data[run_name] = round(1000 * (timer_algo_end - timer_algo_start))
 
 
+@contextlib.contextmanager
+def busy_wait(enabled):
+    if enabled:
+        katana.set_busy_wait()
+    yield
+    if enabled:
+        katana.set_busy_wait(0)
```

Comment:
I wonder if we should get ride of the other API and only provide a context manager like this. Hmm.